lib/config/schemas/gateway.config.json

```diff
@@ -95,15 +95,29 @@
         "type": "object",
         "properties": {
           "url": {
-            "type": "string"
+            "type": "string",
+            "format": "uri"
           },
           "urls": {
             "type": "array",
             "items": {
-              "type": "string"
+              "type": "string",
+              "format": "uri"
             }
           }
-        }
+        },
+        "oneOf": [
+          {
+            "required": [
```

Comment:
do you plan next PR with "pattern" support? will it be `url`  property that is a pattern (then no need for format) or it will be a separate property ?

Reply:
I'm not entirely sure I get what you mean, could you elaborate?

Reply:
I mean what is the point in having `url` if soon you'll add another property like `pattern` that will be used to build url from expression. like `$1$2`
if you assume it will be same property `url` then it is ok to make it required. otherwise, you would soon have to revert this schema change

Reply:
As far as I know, there's no short term plan to make that thing a pattern yet. We've been discussing this with Al but I think we have other priorities. Either way, I'd try to keep the new behavior in this same property, rather than adding a new one.